app/index.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const { body, validationResult, check } = require("express-validator");
+const {
+  autoSuggest,
+  addDataHandler,
+  getDateHandler,
+  updateDataHandler,
+  deleteDataHandler,
+  autoSuggestHandler,
+} = require("../controllers/handlers");
+
+const app = express();
+
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.use(express.static("../public"));
+
+app.post(
+  "/submit",
+  [
+    check("age")
+      .isInt({ min: 4, max: 130 })
+      .withMessage("age must be between 4 and 130 "),
+    validatePassword,
+  ],
+  (req, res) => {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+      res.status(400).send(errors);
+    }
+    const acknowleagement = addDataHandler(req.body);
+    res.send(acknowleagement);
+  }
+);
+app.post("/get", (req, res) => {
```

Comment:
```suggestion
app.post("/user/:id", (req, res) => {
```
Route name should be user. Should rather use a path parameter to get the user.